packages/client/src/forms/configuration/default.ts

```diff
@@ -3375,7 +3372,8 @@ export const registerForms: IDefaultRegisterForms = {
                 conditionals: [
                   {
                     action: 'hide',
-                    expression: 'values.causeOfDeathEstablished !== "true"'
+
+                    expression: 'values.causeOfDeathEstablished !== true'
```

Comment:
Is there a need for the empty line here?

Reply:
I will remove it